todoList.test.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @jest-environment jsdom
```

Comment:
Please add the `npm run test` command in the README file.

Reply:
added npm run test command to Readme File